### DISCUSSION TITLE
为什么侧边栏会出现两个“References”选项卡

### DISCUSSION THREAD

**Discussion by xinwuchn** [MuiseDestiny/zotero-reference#109] [Q&A]
疑问如图。
<img width="362" alt="image" src="https://user-images.githubusercontent.com/62836462/227894065-d8ccac28-e4ee-4ba1-9162-8ac61ced50dd.png">

**Comment by caiweibing666** [comment]
同问
![image](https://user-images.githubusercontent.com/129829871/229707028-e231c8e0-a1dd-45c3-a5d9-df7f4b5b1059.png)

**Reply by 6000-ai** [reply]
确实会这样，同问加1

**Reply by MuiseDestiny** [reply]
插件冲突

**Reply by 6000-ai** [reply]
请问该怎么解决啊，咋看是和哪个插件冲突了

**Comment by caiweibing666** [comment]
更新后侧栏不冲突了，但是分栏按钮有时会不显示，或者多显示
![image](https://github.com/MuiseDestiny/zotero-reference/assets/129829871/08cdb5bc-dda5-4efc-a722-9905b7408ef6)

**Reply by MuiseDestiny** [reply]
重启Zotero。

**Reply by caiweibing666** [reply]
重启还是会出现这个现象

**Reply by MuiseDestiny** [reply]
卸载重装

**Reply by caiweibing666** [reply]
重装插件还是zotero呢？重装插件还是存在

**Reply by MuiseDestiny** [reply]
下载最新版插件卸载老的，安装新的，然后关闭zotero软件，再打开zotero软件。

**Reply by caiweibing666** [reply]
是这样操作，还是会有这现象😓

**Reply by MuiseDestiny** [reply]
准备todesk远程吧
